apps/sms/js/thread_list_ui.js

```diff
@@ -394,19 +395,24 @@ var ThreadListUI = {
   },
 
   markReadUnread: function thlui_markReadUnread(selected, isRead) {
-    selected.forEach((id) => {
+    var filterIds = (id) => {
```

Comment:
I said in my previous review to rename this to something like "markVisuallyAndFilterMarkable" to make it clear that it's marking visually messages as read as well as filtering.